bin/config.py

```diff
@@ -70,29 +76,49 @@ def set_default_args():
         'check_deterministic',
         'clean',
         'clean_generated',
+        'cleanup_generated',
```

Comment:
The list can be a global const, next to default_args.

Reply:
Done, see 87b84ca